libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/NavigatorLoader.kt

```diff
@@ -27,17 +30,27 @@ internal object NavigatorLoader {
     fun createNavigator(
         deviceProfile: DeviceProfile,
         navigatorConfig: NavigatorConfig,
-        tilesConfig: TilesConfig
+        tilesConfig: TilesConfig,
+        historyDir: String?
     ): NativeComponents {
         val config = ConfigFactory.build(
             settingsProfile(deviceProfile),
             navigatorConfig,
             deviceProfile.customConfig
         )
         val runLoopExecutor = RunLoopExecutorFactory.build()
-        val historyRecorder = null
-        val cache = CacheFactory.build(tilesConfig, config, historyRecorder)
 
+        val historyRecorder = if (historyDir != null) {
+            HistoryRecorderHandle.build(historyDir, config)
+        } else {
+            null
+        }
+
+        val cache = CacheFactory.build(tilesConfig, config, historyRecorder)
+        LoggerProvider.logger.i(
+            Tag("MbxHistoryRecorderOptions"),
+            Message("navigator created")
+        )
```

Comment:
Good to remove?

Reply:
In any case, this is not `HistoryRecorderOptions`, right?

Reply:
Yep 👍 . sorry i didn't do a better cleanse here, thanks for the rush help 😄 

removed